### ISSUE TITLE
"Failed to launch JVM on startup"

### ISSUE THREAD

**Issue by Spacewolf4** [crschnick/pdx_unlimiter#113]
Running fully updated Win11 on a new laptop. Have installed Pdx-U from latest installer after installing everything else needed. I click the desktop icon, a dialog box appears, stating "Failed to launch JVM on startup". Happens regardless of how often I do a clean install. How do I fix this?
![Screenshot 2023-12-13 074124](https://github.com/crschnick/pdx_unlimiter/assets/138301634/47990ac1-4320-48cf-bdbb-732a456d29c5)

**Comment by crschnick** [comment]
Hmm ... can you try downloading the latest standalone zip version, extract it, and run `runtime\\bin\\pdxu.bat` in it?

**Comment by crschnick** [comment]
Yeah the .exe it's just a small launcher to run the .bat file. Sadly I don't have a lot of control over the generated .exe, so I can't really tell you what went wrong there.

Is this your first time running pdxu or has it worked previously somewhere on another system?

**Comment by crschnick** [comment]
If more reports of this issue come in, I could file a Java bug report for that. That is all I can do about this.

**Comment by thandley** [comment]
I have encountered this issue today, after using the windows installer version successfully for over a year.

It happened after I tried to load my latest CK3 save and an unrelated bluescreen on Win 11 crashed the game. When I rebooted I can't get the installed version to start.

The standalone one works.

**Comment by crschnick** [comment]
@thandley You can try deleting C:\Users\<User>\AppData\Local\Programs\Pdx-Unlimiter\app\ and relaunching pdxu. That should trigger a new download.

**Comment by crschnick** [comment]
Although that definitely indicates that something on your hard drive has been corrupted if it worked before. The bluescreen might be related to that. Personally I would look into checking the disk for errors and preparing a replacement, you don't want it to die on you unexpectedly.

**Comment by thandley** [comment]
That worked, thanks!
